src/main/java/seedu/address/model/shift/ShiftTime.java

```diff
@@ -9,14 +10,9 @@
  */
 public class ShiftTime {
 
-    enum TimeValue {
-        AM,
-        PM
-    }
-
     public static final String MESSAGE_CONSTRAINTS =
-            "Shift time should only contain one of the following values: AM, PM";
-    public final TimeValue time;
+            "Shift time should only contain one of the following values: AM, PM, FULL";
```

Comment:
I'm not sure if the `FULL` value should be implemented into the code itself, or should it just be an argument that we accept that from user input that is essentially a wrapper for creating both an `AM` and a `PM` unavailability. Not sure if this might result in duplicates when creating shifts of `AM` and of `FULL` of the same day?

Reply:
Made the change to convert FULL to AM and PM unavailabilities